frontend/src/AlertsManagement/AlertsContent.test.js

```diff
@@ -136,12 +136,27 @@ describe("fetch alerts", () => {
   // Expected allAlerts, unchecked, and checked.
   const expectedAlerts = new Map();
   expectedAlerts.set(0, {
-    timestamp: "Sun Dec 08 2019",
-    anomalies: 2,
+    timestampDate: "Sun Dec 08 2019",
```

Comment:
since you have a day of the week here, and the timestamp from datastore does not have a day of the week, i was wondering ig the timestamp object in javascript need to have a day of the week?